curtisgr/99bottles-python
# Update tests

- Change from unittests to pytest
- Use skip instead of commented code
- Add a Makefile to:
  - Check format
  - Make format
  - Run the tests
